portal-impl/src/com/liferay/portal/servlet/filters/util/CacheFileNameGenerator.java

```diff
@@ -27,10 +27,11 @@
 public class CacheFileNameGenerator {
 
 	public static String getCacheFileName(
-		HttpServletRequest httpServletRequest, String cacheName) {
+		HttpServletRequest httpServletRequest) {
```

Comment:
@julschong For this, please do:
1) pass in CacheKeyGenerator rather than a cacheName
2) apply to your new API
This way it would be clear that you need a default key generator.